src/components/SearchableDropdownList.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+
+import SearchField from './SearchField';
+import SearchableDropdownListItem from './SearchableDropdownListItem';
+
+const SearchableDropdownList = ({ places, handleListItemClick }) => {
+  const [filteredSearches, setFilteredSearches] = useState([]);
+  const [searchWord, setSearchWord] = useState('');
+
+  useEffect(() => {
+    setFilteredSearches(places);
+  }, [places]);
+  
+  const handleChange = (e) => {
+    setSearchWord(e.target.value);
```

Comment:
you can either destructure value right within params: `... = ({ target: { value }}) => ...`, 
or create a `const value = event.target.value`